plugins/convertPathData.js

```diff
@@ -1064,6 +1090,22 @@ function isCurveStraightLine(data) {
   return true;
 }
 
+/**
+ * Calculates the sagitta of an arc if possible.
+ *
+ * @type {(data: number[]) => number | undefined}
+ * @see https://wikipedia.org/wiki/Sagitta_(geometry)#Formulas
+ */
+function calculateSagitta(data) {
+  if (data[3] === 1) return undefined;
+
+  const [rx, ry] = data;
+  if (Math.abs(rx - ry) > error) return undefined;
+  const chord = Math.sqrt(data[5] ** 2 + data[6] ** 2);
```

Comment:
Related: https://github.com/svg/svgo/pull/1913#issuecomment-1871354356

Welcome to comment there too if you're interested. I forgot to make the change, but in another discussion, the conclusion was also to `Math.hypot` instead of `Math.sqrt(Math.pow, Math.pow)`. I'll update instances of this next time I'm working on SVGO.

Reply:
As far as I can see, all the reasons were mentioned there. Notably, hypot has better precision which is more crucial for SVGO. Aside for being more convenient.